doc/sphinx/run.rst

```diff
@@ -171,157 +171,219 @@ The equilibrium constant of the reaction is then given as
 
 .. math::
 
-   K = \exp(-\Delta_{\mathrm{r}}G / k_B T)
+   K = \exp(-\Delta_{\mathrm{r}}G^{\ominus} / k_B T)
        \quad\text{with}\quad
        \Delta_{\mathrm{r}}G^{\ominus} = \sum_i \nu_i \mu_i^{\ominus}\,.
        \label{Keq}
 
 
 Here :math:`k_B` is the Boltzmann constant, :math:`T` is temperature,
 :math:`\Delta_{\mathrm{r}}G^{\ominus}` standard Gibbs free energy change
-of the reaction, and :math:`\mu_i^{\ominus}` the standard Chemical
-potential of species :math:`i`. Note that thermodynamic equilibrium is
-independent of the direction in which we write Eq.[general-eq]. If it is
-written with left and righ-hand side swapped, the stoichiometric
-coefficients and :math:`\Delta_{\mathrm{r}}G^{\ominus}` attain opposite
-signs, and the equilibrium constant attains the inverse value. Further,
-note that the equilibrium constant :math:`K` in Eq. [Keq] is the
-dimensionless *thermodynamic* equilibrium constant :math:`K_\mathrm{p}`.
-Apparent, concentration-based equilibrium constants can also be found in
-literature. To be used as input for the reaction ensemble, they need to
-be converted to thermodynamic constants as described in texbooks of
-Physical Chemistry. As a special case, all stoichiometric coefficients
-on one side of Eq.[general-eq] can be zero. Such reaction is equivalent
-to exchange with a reservoir, and the simulation in reaction ensemble
-becomes equivalent with the grandcanonical simulation. A simulation in
-the reaction ensemble consists of two types of moves: the reaction move
-and the configuration move. The configuration move is carried out by a
+of the reaction, and :math:`\mu_i^{\ominus}` the standard chemical
+potential (per particle) of species :math:`i`. Note that thermodynamic equilibrium is
+independent of the direction in which we write the reaction. If it is
+written with left and righ-hand side swapped, 
+both :math:`\Delta_{\mathrm{r}}G^{\ominus}` and the stoichiometric
+coefficients attain opposite signs, and the equilibrium constant attains the inverse value. 
+Further, note that the equilibrium constant :math:`K` is the
+dimensionless *thermodynamic, concentration-based* equilibrium constant,
+defined as
+
+.. math::
+
+   K(c^{\ominus}) = (c^{\ominus})^{-\bar\nu} \prod_i (c_i)^{\nu_i}
+
+wher :math:`\bar\nu=\sum_i \nu_i`, and :math:`c^{\ominus}` is the reference concentration,
+at which the standard chemical potential :math:`\Delta_{\mathrm{r}}G^{\ominus}` was determined.
+In practice, this constant is often used with the dimension of :math:`(c^{\ominus})^{\bar\nu}`
+
+.. math::
+
+   K_c(c^{\ominus}) = K(c^{\ominus})\times (c^{\ominus})^{\bar\nu}
+
+A simulation in
+the reaction ensemble consists of two types of moves: the *reaction move*
+and the *configuration move*. The configuration move is performed by a
```

Comment:
I would call it a "configuration changing move"

Reply:
I see no reason for giving it a longer name. But I will modify the subsequent explanation to make it clearer.